### ISSUE TITLE
Import pyfontaine char/glyph lists

### ISSUE THREAD

**Issue by davelab6** [graphicore/charset-inspector#1]
This is a rewrite of https://github.com/davelab6/pyfontaine - good!

pyfontaine has some glyph lists, and various methods, but its all CLI. 

I'd be happy for you to port everything in pyfontaine here, and shut that project down.

**Comment by graphicore** [comment]
Can expand "everything in pyfontaine" into a list?

There are already some todos in the [README](https://github.com/graphicore/charset-inspector#todo) append there? Or we bring the todos into the issue tracker.

Do you still need a CLI?

Admittedly, I'm not very active in this project. But I had some ideas to use it elsewhere ... What I'm saying is that it's not dead, but development is slow.

**Comment by graphicore** [comment]
Yeah, the Commander fork of Metapolator plus [helmsman](https://github.com/mattmcmanus/node-helmsman) make a nice pair. Though, we can also use the not forked Commander.

**Comment by davelab6** [comment]
> Can expand "everything in pyfontaine" into a list?

Will do

> There are already some todos in the README append there? Or we bring the todos into the issue tracker.

Latter better imho

> Admittedly, I'm not very active in this project. But I had some ideas to use it elsewhere ... What I'm saying is that it's not dead, but development is slow.

But this has more future than pyfontaine :)
